### PR TITLE
update the content based on Grammarly corrects

### PR DESCRIPTION
**Pull request by MajedBojan**
While working on supporting `ar README` I found some paragraphs that need to be updated based on [Grammarly](app.grammarly.com)

ar README will be raised in independent PR once completed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -548,7 +548,7 @@ pick 4b6e19a The second to last commit
 pick f4037ec The last commit
 ```
 
-This tells rebase that you want to edit your third last commit and keep the other two unaltered. Then you'll save (and close) the editor. Git will then start to rebase. It stops on the commit you want to alter, giving you the chance to edit that commit. Now you can apply the changes which you missed applying when you initially committed that commit. You do so by editing and staging them. Afterwards you'll run
```
Comment (RichardLitt) [inline README.md:551]: Afterwards is fine?

File: README.md [hunk 2]
```diff
@@ -1184,7 +1184,7 @@ origin/HEAD set to master
 
 ### I made changes on the wrong branch
 
-You've made uncommitted changes and realise you're on the wrong branch. Stash changes and apply them to the branch you want:
```
Comment (RichardLitt) [inline README.md:1187]: I prefer British spelling. Can you not change these?

File: README.md [hunk 3]
```diff
@@ -227,7 +227,7 @@ Note that now you have two remotes.
 
 From origin, you can read and write. From upstream, you can only read.
 
-When you've finished making whatever changes you like, push your changes (normally in a branch) to the remote named `origin`. If you're on a branch, you could use `--set-upstream` to avoid specifying the remote tracking branch on every future push using this branch. For instance:
```
Comment (RichardLitt) [inline README.md:230]: Why the dash?

File: README.md [hunk 4]
```diff
@@ -37,12 +37,12 @@ All commands should work for at least git version 2.13.0. See the [git website](
     - [I committed with the wrong name and email configured](#i-committed-with-the-wrong-name-and-email-configured)
     - [I want to remove a file from the previous commit](#i-want-to-remove-a-file-from-the-previous-commit)
     - [I want to delete or remove my last commit](#i-want-to-delete-or-remove-my-last-commit)
-    - [Delete/remove arbitrary commit](#deleteremove-arbitrary-commit)
+    - [Delete/remove an arbitrary commit](#deleteremove-arbitrary-commit)
```
Comment (RichardLitt) [inline README.md:40]: Rerun Tocdoc. This doesn't change the hyperlinks.

**Review by RichardLitt — CHANGES_REQUESTED** [review]
Pretty good. Some small changes.

File: README.md (hunk 1 above)
Comment (RichardLitt) [reply README.md:551]: Well that's silly.

File: README.md (hunk 3 above)
Comment (RichardLitt) [reply README.md:230]: This is unnecessary.

File: README.md (hunk 1 above)
Comment (MajedBojan) [reply README.md:551]: @RichardLitt  Do I keep it `Afterwards you'll run` and update the PR

File: README.md (hunk 3 above)
Comment (RichardLitt) [reply README.md:230]: Remove this change.

File: README.md (hunk 1 above)
Comment (RichardLitt) [reply README.md:551]: I'd prefer it. I would go back to Grammarly and tell them this is a useless change.

File: README.md [hunk 5]
```diff
@@ -1914,7 +1914,7 @@ To set an email address that will be associated with each history marker:
 git config --global user.email “[valid-email]”
 ```
 
-### I want to add command line coloring for Git
+### I want to add command-line coloring for Git
```
Comment (RichardLitt) [inline README.md:1917]: Also useless.

**Comment by RichardLitt** [comment]
Thanks, @MajedBojan. Gentle ping. :)

**Review by alexaguilar89 — APPROVED** [review]
Ok a ir changes

**Comment by RichardLitt** [comment]
Closing due to lack of response. Sorry if I came off as too harsh, and I hope this wasn't an Octoberfest PR which needed to be merged by November for you to feel valued.

**Pull request closed without merging**